test/test_images/sendtracker/sendtracker.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	"github.com/google/knative-gcp/pkg/kncloudevents"
+)
+
+const (
+	brokerURLEnvVar = "BROKER_URL"
```

Comment:
I recommend allowing K_SINK in addition to or instead of BROKER_URL. That allows SinkBinding to be used.

Reply:
Yep.  On my list prior to commit.  Put it up to share with people, not at all ready for merge(small tweaks like that)